src/api/authApi.ts

```diff
@@ -50,4 +50,20 @@ export const authApi = {
       });
     return response;
   },
+
+  registrationClient: async (data: {} ) => {
```

Comment:
It feels like data argument should have some type, not be just empty object

Reply:
that is it is necessary to register the type for each element of object ? )

Reply:
I'd say yes, otherwise typescript won't be able to check if you pass correct arguments to this funciton